### DISCUSSION TITLE
Run Simulation Problem

### DISCUSSION THREAD

**Discussion by Goblinking13** [ichi4096/vivado-on-silicon-mac#60] [Q&A]
Hello. I downloaded Vivado 2022.2 by installation instruction, everything was right, but when i try to run simulation got this problem:
ERROR: [USF-XSim-62] 'compile' step failed with error(s) while executing '/home/user/Xilinx/vhdl_02/vhdl_02.sim/sim_1/behav/xsim/compile.sh' script. Please check that the file has the correct 'read/write/execute' permissions and the Tcl console output for any other possible errors or warnings.
Is there any ways to solve it? i have apple M2, macOS Ventura 13.0.
Thanks for any help

**Comment by ichi4096** [comment]
Are there any other errors messages along with that one? 

Could You open a terminal inside the Linux container and type in `ls -lah /home/user/Xilinx/vhdl_02/vhdl_02.sim/sim_1/behav/xsim/compile.sh` and report the output?

**Reply by Goblinking13** [reply]
In Error messages also have: '[Vivado 12-4473] Detected error while running simulation. Please correct the issue and retry this operation.'. 
Terminal output: '-rwxrwxrwx 1 user user 802 Oct  9 10:33 /home/user/Xilinx/vhdl_02/vhdl_02.sim/sim_1/behav/xsim/compile.sh'.

**Reply by ichi4096** [reply]
So the compile.sh file is executable. I don't think I can help You further other than advise You to try reinstalling Docker and redownloading my repo.

If You encounter errors mentioning "VMAllocationTracker.cpp", those are/were bugs in Apple's Rosetta implementation.
